lean4/src/putnam_2019_a4.lean

```diff
@@ -1,12 +1,12 @@
 import Mathlib
 open BigOperators
 
-open Topology Filter
+open MeasureTheory Metric Topology Filter
 
 abbrev putnam_2019_a4_solution : Prop := sorry
 -- False
 theorem putnam_2019_a4
-(fint : ((EuclideanSpace ℝ (Fin 3)) → ℝ) → Prop)
-(hfint : ∀ f : (EuclideanSpace ℝ (Fin 3)) → ℝ, fint f = (∀ S : EuclideanSpace ℝ (Fin 3), (∫ x in {p : EuclideanSpace ℝ (Fin 3) | dist p S = 1}, f x) = 0))
-: (∀ f : (EuclideanSpace ℝ (Fin 3)) → ℝ, (Continuous f ∧ fint f) → (∀ x : EuclideanSpace ℝ (Fin 3), f x = 0)) ↔ putnam_2019_a4_solution :=
-sorry
+    (P : (EuclideanSpace ℝ (Fin 3) → ℝ) → Prop)
+    (P_def : ∀ f, P f ↔ ∀ C, integral μH[2] ((sphere C 1).restrict f) = 0) :
```

Comment:
Isn't there a notation for integrating with respect to a measure?

Reply:
Oh indeed there is, so what I have written is defeq to `∫ x, (sphere C 1).restrict f x ∂μH[2]`. However I now realise that this is equal to `∫ x in sphere C 1, f x ∂μH[2]` (via `MeasureTheory.integral_subtype_comap` and some messing).